tests/integration/test_storage_s3/test.py

```diff
@@ -355,8 +355,18 @@ def run_s3_mock(cluster):
 
 
 def test_custom_auth_headers(cluster):
-    ping_response = cluster.exec_in_container(cluster.get_container_id('resolver'),
+    for i in range(100):
+        try:
+            ping_response = cluster.exec_in_container(cluster.get_container_id('resolver'),
```

Comment:
This test already fixed there https://github.com/ClickHouse/ClickHouse/pull/17299